src/common/components/ui/EventsModal.css

```diff
@@ -0,0 +1,8 @@
+.events-modal .pf-c-modal-box__body {
```

Comment:
instead of this selector, it would be better to wrap the `EventListFetch` in div with className, something like
```js
<div className="events-modal__body">
  <EventListFetch
    hostId={hostId}
    cluster={cluster}
    entityKind={entityKind}
    onFetchEvents={onFetchEvents}
  />
</div>
```